src/Microsoft.ML.ImageAnalytics/ImageLoader.cs

```diff
@@ -238,8 +238,10 @@ private Delegate MakeGetterImageDataViewType(DataViewRow input, int iinfo, Func<
                                 path = Path.Combine(_parent.ImageFolder, path);
 
                             // to avoid locking file, use the construct below to load bitmap
-                            using(var bmpTemp = new Bitmap(path))
-                                dst = new Bitmap(bmpTemp) { Tag = path };
+                            var bytes = File.ReadAllBytes(path);
```

Comment:
This is interesting. Can you please help me understand why the change was necessary?

Reply:
The first change was not preserving the original pixel encoding (format) from the file. It looks like the copy constructor was modifying the format.  The second iteration preserves original encoding by moving all the bytes into the byte stream, so only underlying media is changed (memory vs file). See https://stackoverflow.com/questions/4803935/free-file-locked-by-new-bitmapfilepath for more details